lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -186,13 +250,45 @@ define([
                             }
                         }]
                     });
-                    assertTeamDetails(view, 1);
+                    assertTeamDetails(view, 1, true);
 
                     expect(view.$('.invite-link-input').length).toBe(1);
 
                     view.$('.invite-link-input').click();
                     expect(view.selectText).toHaveBeenCalled();
                 });
+                it('can leave team successfully', function() {
```

Comment:
Can we add jasmine test for error scenario, i mean what happens when ajax call fails.